docs/content/index.md

```diff
@@ -1,11 +1,16 @@
-# ProjectScaffold
+##What is ProjectScaffold?
+ProjectScaffold helps you get started with a new F# .NET/Mono project solution with everything needed for successful organising of code, tools and publishing. 
```

Comment:
Sure, but wouldn't it be clearer if we have F# now and other langs added when they become available.